src/components/molecules/LoadingPage/LoadingPage.tsx

```diff
@@ -27,30 +36,35 @@ const quotes = IS_BURN
     ]
   : ["Loading..."];
 
-export const LoadingPage = () => {
+const _LoadingPage = () => {
   const [quote, setQuote] = useState("Loading...");
 
   useInterval(
-    () => {
+    useCallback(() => {
       setQuote(quotes[Math.floor(Math.random() * quotes.length)]);
-    },
+    }, []),
     IS_BURN ? 1000 : undefined
   );
 
+  const { isShown: animate, show: startAnimation } = useShowHide();
+  //this prevents "flickering" of the animation on page load
+  useTimeoutFn(startAnimation, LOADING_PAGE_ANIMATION_DELAY);
+
   return (
-    <div className="loading-screen">
-      <div className="loading-content">
-        <div className="burningman-loading-container">
-          <span className="loading-sparkle-1" />
-          <span className="loading-sparkle-2" />
-          <div className="burningman-loading">
-            <div className="burningman-loading-anim" />
-          </div>
+    <div className="LoadingPage">
+      <div className="LoadingPage__wrapper">
+        <div className="LoadingPage__logo-container">
+          <DiamondSvg
+            className={classNames("LoadingPage__diamond", {
+              "LoadingPage__diamond--animation": animate,
```

Comment:
The animation should be always executed, no? You can remove the conditional state and always play it.

Reply:
explanation here https://github.com/sparkletown/sparkle/pull/2005#discussion_r692679172